README.md

```diff
@@ -15,56 +15,90 @@ Keyword Management System (KMS) is a application for maintaining keywords (scien
 
 To install the necessary components, run:
 
-```bash
-    npm install
+```
+npm install
```

Comment:
Specify which version to use

Reply:
it is actually mentioned, check .nvmrc for correct version

Reply:
Yes, I think it is fine, it is mentioned above to check that .nvmrc file to determine the version.